draft-ietf-quic-http.md

```diff
@@ -459,8 +459,8 @@ An HTTP message (request or response) consists of:
 3. optionally, the trailer field section, if present, sent as a single HEADERS
    frame.
 
-Header and trailer field sections are described in Section 5 of {{!SEMANTICS}};
-the payload body is described in Section 7.3.3 of {{!SEMANTICS}}.
+Header and trailer field sections are described in Section 5.4 of {{!SEMANTICS}};
```

Comment:
Section 5.4 is about headers only, not trailers.  This may need to explicitly reference 5.4 and 5.6 to get the equivalent context.